test-utils/src/main/java/apoc/util/TestContainerUtil.java

```diff
@@ -171,6 +200,13 @@ private static Neo4jContainerExtension createNeo4jContainer(
                     + "the docker container because only apoc core comes bundled in those");
         }
 
+        String cypherVersionSetting = cypherVersion == null
+                ? System.getenv()
+                        .getOrDefault(
+                                "CYPHER_VERSION",
+                                Util.getCypherVersion(GraphDatabaseInternalSettings.CypherVersion.Cypher25))
```

Comment:
Note: I will convert this back to 5 before merge, but wanted to show the tests are all fixed for Cypher 25 :D